all.js

```diff
@@ -369,6 +401,85 @@ function getToolsFolder() {
 }
 
 /**
+ * Finds SSH key
+ * @returns {string}
+ */
+function findSshKey() {
+  if (config.device_key_path) {
+    if (fileExistsSync(config.device_key_path)) {
+      return fs.readFileSync(config.device_key_path, { encoding: 'ascii'});
+    }
+  }
+
+  return false;
+}
+
+/**
+ * Loads combined config
+ * @param {string} postfix  - postfix appended to global config filename
+ * @returns {object}
+ */
+function readCombinedConfig(postfix) {
+  var config = {};
+  var globalConfig = readGlobalConfig(postfix);
+  var localConfig = require(process.cwd() + '/config.json');
+  var combinedConfig = Object.assign(config, globalConfig, localConfig);
+  return combinedConfig;
+}
+
+/**
+ * Get loaded config
+ * @returns {object}
+ */
+function getConfig() {
+  return config;
+}
+
+/**
+ * Loads selected config from user folder
+ * @param {string} postfix  - postfix appended to config filename
+ * @returns {object}
+ */
+function readGlobalConfig(postfix) {
+  var filename = getToolsFolder() + '/config-' + postfix + '.json';
+
+  if (fileExistsSync(filename)) {
+    return require(filename);
+  }
+
+  return {};
+}
+
+/**
+ * Writes selected config to user folder
+ * @param {string} postfix  - postfix appended to config filename
+ * @param {object} config   - config object
+ */
+function writeGlobalConfig(postfix, config) {
+  fs.writeFileSync(getToolsFolder() + '/config-' + postfix + '.json', JSON.stringify(config, null, 2));
+}
+
+/**
+ * Updates or creates global config file
+ * @param {string} postfix    - postfix appended to config filename
+ * @param {object} template   - config template
+ * @returns {object}
+ */
+function updateGlobalConfig(postfix, template) {
```

Comment:
this is a very good suggestion, i was thinking of this myself, so it proves it makes sense. it should be something "config file '...' was created / updated" or something like this. I think this should be done asap.